hazelcast/src/main/java/com/hazelcast/spi/impl/operationparker/impl/OperationParkerImpl.java

```diff
@@ -216,7 +216,11 @@ private void invalidateWaitingOps(String callerUuid) {
         }
     }
 
-    // This is executed under partition migration lock!
+    /**
+     * Invoked on the migration destination to invalidate all parked operations for the migrated partition and send a
```

Comment:
`Invalidates all parked operations for the migrated partition and sends a {@link PartitionMigratingException} as a response. Invoked on the migration destination. This is executed under partition migration lock!`